packages/jest-environment/jest.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import("jest").Config} */
+module.exports = {
+  testEnvironment: './dist',
+  setupFilesAfterEnv: ['./jest.setup'],
+  transform: {
+    '\\.(js|jsx|ts|tsx)$': [
+      '@swc/jest',
```

Comment:
not sure why is this necessary; we're already using ts-jest for transpiling:
https://github.com/vercel/edge-runtime/blob/main/jest.config.ts#L10

is swc solution a faster approach? can we use it in both places?